src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/abstractjavadoc/InputAbstractJavadocCorrectParagraph.java

```diff
@@ -1,5 +1,11 @@
 package com.puppycrawl.tools.checkstyle.checks.javadoc.abstractjavadoc;
 
+/*
+ * Config: AtClauseOrderCheck
+ *
+ * Config: JavadocParagraphCheck
+ */
```

Comment:
Does this mean that this file is still used in multiple test methods?

Reply:
No, it means that the test is testing this file with two configs

Reply:
https://github.com/checkstyle/checkstyle/blob/8dd007530e974bb5d0abc0d947b4bb86ad8d4a2a/src/test/java/com/puppycrawl/tools/checkstyle/checks/javadoc/AbstractJavadocCheckTest.java#L140-L143